crates/sui-config/src/genesis_config.rs

```diff
@@ -230,6 +230,19 @@ impl GenesisConfig {
         )
     }
 
+    pub fn for_local_testing_with_addresses_and_faucet(addresses: Vec<SuiAddress>) -> Self {
+        let mut custom_genesis = Self::custom_genesis_with_addresses(
+            DEFAULT_NUMBER_OF_AUTHORITIES,
+            addresses,
+            DEFAULT_NUMBER_OF_OBJECT_PER_ACCOUNT,
+        );
+        custom_genesis.accounts.push(AccountConfig {
+            address: None,
+            gas_amounts: vec![DEFAULT_GAS_AMOUNT; DEFAULT_NUMBER_OF_OBJECT_PER_ACCOUNT],
```

Comment:
what is this for?

Reply:
It creates the genesis config that is used for loading the validator nodes and various other set ups.

Reply:
Removed it, it was unnecessary